.github/workflows/ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    # every Monday
+    - cron:  '0 0 * * 1'
+
+jobs:
+  build:
+    runs-on: [windows-latest]
+    strategy:
+      fail-fast: false
+      matrix:
+        ROSDISTRO: [foxy]
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        path: src
+    - name: Install
+      shell: cmd
+      run: |
```

Comment:
This needs to use the Github action.

Reply:
Our goal is to fix the ros-tooling github action when it breaks on Windows.
https://github.com/ros-tooling/action-ros-ci